CHANGELOG.md

```diff
@@ -1,3 +1,69 @@
+# 0.0.114 - XXX - "Faster Async BOLT12 Retries"
+
+## API Updates
```

Comment:
`KeysInterface::get_node_secret` was also removed. I think that sorta breaks our message signing API as well, since that takes a node secret.